packages/astro/src/content/utils.ts

```diff
@@ -492,7 +492,7 @@ export async function getEntrySlug({
 		contents = await fs.promises.readFile(fileUrl, 'utf-8');
 	} catch (e) {
 		// File contents should exist. Raise unexpected error as "unknown" if not.
-		throw new AstroError(AstroErrorData.UnknownContentCollectionError, { cause: e });
```

Comment:
I still don't understand why the second argument is `undefined` 🤔

Reply:
That is the message.
```ts
const error = new Error("error message", { cause })
```
AstroError's constructor overrides the message (`this.message = errorData.message`) anyway so it might be possible to remove the option altogether.

Reply:
I was a bit worried but it was relatively simple.